docs/docs/examples/guides/jtc-guide.md

```diff
@@ -196,28 +196,49 @@ read through your URDF.
 
 ### Using JTC's `Set trajectory` service
 
-The service currently accepts 5 variables, namely:
+The service currently accepts 6 variables, namely:
 
 - `times_from_start`: a list of timestamps (in seconds) measured from the start, indicating when JTC should reach each
 frame.
+- `durations`: a list of absolute durations (in seconds) that correspond to each waypoint.
 - `frames`: names of the Cartesian frames to reach with the robot end effector 
 - `joint_positions`: names of joint positions that the robot should achieve
 - `blending_factors`: factors in [0.0, 1.0] indicating the amount of curving allowed between 2 consecutive waypoints.
-The first and last segment are currently not considered in blending, therefore, the vector's size needs to be equal to
-the number of frames or joint positions **minus** 2. **The default blending factors are all set to 0.0.**
+The default blending factors are all set to 0.0. The last trajectory segment is not considered in blending, therefore,
+the vector's size needs to be equal to the number of frames or joint positions **minus** 1 or simply contain a single
+value to be applied to all waypoints that support blending.
 - `blending_samples`: the number of samples **(minimum 10; default 50)** to be used when generating the blended
 trajectory. If you find that your blended trajectory is not smooth enough, consider increasing this number.
 
-While `times_from_start` is always required, only one of `frames` and `joint_positions` can be used at a time. The
-former is a vector of Cartesian frames from which an Inverse Kinematics (IK) solver will compute the joint positions
-that the robot should reach, while the latter refers to joint positions that have been recorded and are being published
-as named joint positions.
+At least one of `times_from_start` or `durations` and one of `frames` and `joint_positions` are always required. 
```

Comment:
One sentence in the direction of "the payload can only contain one of frames and joint positions and either times from start or durations" would be great